### DISCUSSION TITLE
How to change page title {site_name}

### DISCUSSION THREAD

**Discussion by SingingFoot** [squidfunk/mkdocs-material#7719] [Q&A]
I noticed that it is not possible to change the title of the page completely. If you change it in the file mkdocs.yml:

```yml

# Project information
site_name: My_site

```
only the second part changes. The first part {Material for MkDocs} remains unchanged, so you get such site_name:

**Material for MkDocs - My_site**

. I tried to find where this first part is and got over 340 results. A cursory look at various most likely involved files (such as main.html, home.html etc.) did not yield anything. Where can this first part be hidden?

**Comment by SingingFoot** [comment]
I found it in 2 minutes!

![Screenshot 2024-11-19 at 10 09 30](https://github.com/user-attachments/assets/6b797d0e-6c73-4a03-a2b3-e5d3c3253e41)

**Reply by kamilkrzyskow** [reply]
GitHub search isn't really the best at this sort of stuff 😅 In the docs search, it's the second result:
- https://squidfunk.github.io/mkdocs-material/reference/?q=title
- https://squidfunk.github.io/mkdocs-material/reference/#setting-the-page-title

Also the page.meta.title and config.site_name are merged in the base.html template:
https://github.com/squidfunk/mkdocs-material/blob/a08809a0f3e43187aadb9017204171bfa40b3548/src/templates/base.html#L89-L98
